pkg/pulumiyaml/sort.go

```diff
@@ -234,3 +242,7 @@ func checkUniqueNode(intermediates map[string]graphNode, node graphNode) syntax.
 	}
 	return diags
 }
+
+func isPotentialConfig(name string) bool {
+	return len(strings.Split(name, ":")) == 2
```

Comment:
This is simpler and faster.
```suggestion
	return strings.Count(name, ":") == 2
```